lib/api.ts

```diff
@@ -85,6 +88,24 @@ export async function getContributorsSlugs(): Promise<{ file: string }[]> {
   return contributorSlugs;
 }
 
+async function getGithubDiscussions(githubHandle: string) {
+  const response = await fetchGithubDiscussionForUser(githubHandle);
+  const discussions = await response.map((discussion: ParsedDiscussion) => {
+    const title =
+      discussion.author === githubHandle
+        ? `Started a Discussion`
+        : `Commented on a Discussion`;
+    return {
+      type: "discussion",
+      title: title,
+      time: discussion.time,
+      link: discussion.link,
+      discussion: discussion,
+    };
+  });
```

Comment:
Let's keep sentence constructions in the place where the page is rendered.

```suggestion
    return {
      type: "discussion",
      title: discussion.author === githubHandle ? "authored" : "commented",
      time: discussion.time,
      link: discussion.link,
      discussion: discussion,
    } as const;
  });
```

Reply:
I am planning that, We have to give a points for discussions.
So I there will be three terms based on that we have to give a point
1. Comment_created in discussions
2. Discussion_created (author)
3. Who answered that discussion

So My plan is for comment_created in the discussions we can merge that with comment_created as already we have that field

![image](https://github.com/user-attachments/assets/bea0d1c5-09da-44bc-aca5-ca7fc29cd80d)
![image](https://github.com/user-attachments/assets/27c69603-9cf8-4a10-a975-729f2f75e05c)